packages/venia-ui/lib/queries/getStoreConfigData.graphql

```diff
@@ -2,5 +2,10 @@ query storeConfigData {
     storeConfig {
         id
         copyright
+        code
+        locale
+        base_currency_code
+        default_display_currency_code
+        grid_per_page
```

Comment:
The category and searchPage components should each have their own queries for `storeConfig` data and it should be pared down to _just_ the data those components need. Infact it looks like they both only use `grid_per_page` so you should be able to create a much smaller query next to each:

```gql
const getPageSize = gql`
  query getPageSize {
    storeConfig {
      grid_per_page
    }
  }
}
```

Reply:
And you can remove `grid_per_page` from this query.

Reply:
Moved queries for `storeConfig` to respective components